tests/fixture/e2e/ginkgo_test_context.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package e2e
+
+import (
+	"context"
+	"io"
+	"time"
+
+	"github.com/onsi/ginkgo/v2/formatter"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/tests"
+	"github.com/ava-labs/avalanchego/wallet/subnet/primary/common"
+
+	ginkgo "github.com/onsi/ginkgo/v2"
+)
+
+type GinkgoTestContext struct{}
+
+func NewTestContext() *GinkgoTestContext {
+	return &GinkgoTestContext{}
+}
```

Comment:
Why not embed `ginkgo.GinkgoT()` and possible `require.New(t)` here to cut out some of the wrapper code?